cogs/commands.py

```diff
@@ -219,6 +221,55 @@ async def play(self, interaction: ApplicationCommandInteraction, query: str, ind
                                )
                            ] + filter_warnings
                 )
+            case LoadType.SEARCH:
+                options = []
+                result = await self.bot.lavalink.get_tracks(f"{query}")
```

Comment:
Why fetch the search results twice here? We've already fetched then on the code above (results variable)

Reply:
ok my bad
I will be changed in next commits